core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -52,8 +52,11 @@
  */
 public class GlobalProperties {
 
+	private static final String DATABASE_VERSION = "1.2.1";
```

Comment:
Def better than setting as property value, but this is also more something like EXPECTED_DB_VERSION. There should be code elsewhere that decides what to do when the database version does not match up with the expected one.